### PR TITLE
pythonPackages.karton-yaramatcher: 1.1.0 -> 1.1.1

### PR DESCRIPTION
**Pull request by chivay**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 123668` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.karton-yaramatcher</li>
    <li>python39Packages.karton-yaramatcher</li>
  </ul>
</details>

**Review by fabaff — APPROVED** [review]
One change needed

File: pkgs/development/python-modules/karton-yaramatcher/default.nix [hunk 1]
```diff
@@ -24,7 +24,6 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace requirements.txt \
-      --replace "karton-core==4.0.5" "karton-core" \
       --replace "yara-python==4.0.2" "yara-python" \
```
Comment (fabaff) [inline pkgs/development/python-modules/karton-yaramatcher/default.nix:27]: This looks that the change for `yara-python` is [out-dated](https://github.com/CERT-Polska/karton-yaramatcher/blob/master/requirements.txt#L2) and can be removed, too.

**Review by fabaff — CHANGES_REQUESTED** [review]

**Review by jonringer — APPROVED** [review]
LGTM after @fabianhjr 's change

**Comment by chivay** [comment]
Done :+1:

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 123668 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>2 packages built:</summary>  <ul>    <li>python38Packages.karton-yaramatcher</li>    <li>python39Packages.karton-yaramatcher</li>  </ul></details>

**Pull request merged**